prt_partner_tweaks_access/tests/test_partner_tweaks_access.py

```diff
@@ -6,19 +6,23 @@
 
 @tagged("post_install", "-at_install", "standart")
 class TestPartnerTweaksAccess(PartnerTweaksAccessCommon):
-    def test_user_without_tweak_access(self):
+    def test_user_access_difference(self):
         """
-        Check that results differ for a user without tweaks and a user with tweaks.
+        Check the search results are different for a user without access tweaks
+        and a user with access tweaks.
         """
-
-        result_1 = self.partner_model.with_user(self.user_without_tweaks_access).search(
-            []
-        )
-        result_2 = self.partner_model.with_user(self.user_with_tweaks_access).search([])
+        result_without_tweaks = self.partner_model.with_user(
+            self.user_without_tweaks_access
+        ).search([])
+        result_with_tweaks = self.partner_model.with_user(
+            self.user_with_tweaks_access
+        ).search([])
 
         # Ensure that the search results are not equal for the two users
         self.assertNotEqual(
-            result_1, result_2, msg="The search results should not be equal"
+            result_without_tweaks,
```

Comment:
"Same shit different name"(c) ))
Result is not the same. But is it due to our settings or this is just happened because of something else?

Reply:
Got it, I'll put this case away. I tried to display what the user will see after he added restrictions.
As I understand it, it doesn't make sense because other modules can affect the result